general/administration/hardware-acceleration.md

```diff
@@ -49,6 +49,117 @@ The hardware acceleration is available immediately for media playback. No server
 
 Each hardware acceleration type, as well as each Jellyfin installation type, requires different setup options before it can be used. It is always best to consult the FFMpeg documentation on the acceleration type you choose for the latest information.
 
+##Install & Setup for Debain/Ubuntu Nvidia NVENC
+First off we need to figure out what the most up to date driver is for your NVIDIA Card is.
+So for this we need to go to the below link.
+
+[NVIDIA Driver Update](https://www.nvidia.com/Download/index.aspx)
+
+1. You'll need to know what NVIDIA Device you have before searching.
+    -IE. i have an MSI Geoforce GTX 970 for the example of looking up im goin to show you my 1650, thats in the computer im typing this out on but the commands should all show the NVIDIA Geoforce GTX 970 for examples. The only thing that will show my 1650 is this output below.
+        If you dont know what kind of GPU you have you can type in terminal ```lspci``` it will have an output like this
+```00:00.0 Host bridge: Intel Corporation 8th Gen Core 4-core Desktop Processor Host Bridge/DRAM Registers [Coffee Lake S] (rev 07)
+00:01.0 PCI bridge: Intel Corporation Xeon E3-1200 v5/E3-1500 v5/6th Gen Core Processor PCIe Controller (x16) (rev 07)
+00:08.0 System peripheral: Intel Corporation Xeon E3-1200 v5/v6 / E3-1500 v5 / 6th/7th/8th Gen Core Processor Gaussian Mixture Model
+00:14.0 USB controller: Intel Corporation 200 Series/Z370 Chipset Family USB 3.0 xHCI Controller
+00:14.2 Signal processing controller: Intel Corporation 200 Series PCH Thermal Subsystem
+00:16.0 Communication controller: Intel Corporation 200 Series PCH CSME HECI #1
+00:17.0 SATA controller: Intel Corporation 200 Series PCH SATA controller [AHCI mode]
+00:1f.0 ISA bridge: Intel Corporation Device a2ca
+00:1f.2 Memory controller: Intel Corporation 200 Series/Z370 Chipset Family Power Management Controller
+00:1f.3 Audio device: Intel Corporation 200 Series PCH HD Audio
+00:1f.4 SMBus: Intel Corporation 200 Series/Z370 Chipset Family SMBus Controller
+00:1f.6 Ethernet controller: Intel Corporation Ethernet Connection (2) I219-V
+01:00.0 VGA compatible controller: NVIDIA Corporation TU116 [GeForce GTX 1650 SUPER] (rev a1)
+01:00.1 Audio device: NVIDIA Corporation TU116 High Definition Audio Controller (rev a1)
+01:00.2 USB controller: NVIDIA Corporation TU116 USB 3.1 Host Controller (rev a1)
+01:00.3 Serial bus controller [0c80]: NVIDIA Corporation TU116 [GeForce GTX 1650 SUPER] (rev a1)
+```
+This is the ouput you are looking for
+`01:00.3 Serial bus controller [0c80]: NVIDIA Corporation TU116 [GeForce GTX 1650 SUPER] (rev a1)`
+Now check the firmware revision its on by typing ```ndvidia-smi```
```

Comment:
This already requires a driver, in which case you probably don't need the first part of the guide